.gitignore

```diff
@@ -1,3 +1,8 @@
 commands.txt
 
-node_modules
\ No newline at end of file
+node_modules
+
+vm_deploy.sh
```

Comment:
why are you ignoring these files?

Reply:
Im running npm install in the pi, instead of rsyncing over.

The vm_deploy and destroy scripts are for a test in the vm while reimaging the pi.

Reply:
So either remove them from the repo `git rm --cached vm_deploy.sh` or just have them in the repo. It's kinda weird to put them in the gitignore. I'm not sure I follow what you're saying.